install-dev/upgrade/sql/1.7.0.0.sql

```diff
@@ -37,8 +37,6 @@ ALTER TABLE `PREFIX_customer` CHANGE COLUMN `lastname` `lastname` varchar(255) N
 /* Changes regarding modules */
 ALTER TABLE `PREFIX_module` ADD UNIQUE INDEX `name_UNIQUE` (`name` ASC);
 
-DROP TABLE IF EXISTS `PREFIX_modules_perfs`;
```

Comment:
Table was dropped in 1.7.0.0, you don't need to remove it from here, but create it once again in `1.7.7.0.sql` file.